src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -195,11 +195,12 @@ void OutgoingMigration::SyncFb() {
     }
 
     if (cntx_.IsError()) {
-      last_error_ = cntx_.GetError();
-      LOG(ERROR) << last_error_;
-      ThisFiber::SleepFor(1000ms);  // wait some time before next retry
+      auto err = cntx_.GetError();
+      LOG(ERROR) << err.Format();
```

Comment:
why do we print here? if we want to print the error I think it should be inside execution state error reporting